core/src/main/scala/org/apache/spark/serializer/SerializationDebugger.scala

```diff
@@ -169,12 +177,39 @@ private[spark] object SerializationDebugger extends Logging {
             }
             j += 1
           }
-
         }
         i += 1
       }
       return List.empty
     }
+
+    private def visitSerializableWithWriteObjectMethod(
+        o: Object, slotDesc: ObjectStreamClass, stack: List[String]): List[String] = {
```

Comment:
can you use this style:
```
private def theMethod(
    o: Object,
    slotDesc: ObjectStreamClass,
    stack: List[String]): List[String] = {
  ...
}
```
It would also be good to have a java doc that explains what the parameters are, especially since some of them have very terse names e.g. `o`